New reference architecture diagram. (#15693)
cloudflare/cloudflare-docs: content/reference-architecture/diagrams/network/protecting-sp-networks-from-ddos.md

---
title: Protecting ISP and telecommunications networks from DDoS attacks
pcx_content_type: reference-architecture-diagram
weight: 1
meta:
  title: "Protecting ISP and telecommunications networks from DDoS attacks"
---

# Protecting ISP and telecommunications networks from DDoS attacks

## Introduction

> Internet service providers (ISPs) and telecommunications companies (such as T-Mobile or British Telecom) are vulnerable to network DDoS attacks which are often focused on the end customer, i.e. trying to attack a company or remote worker connected to the internet via a broadband internet service provider. Historically to protect these customers, service providers have relied on hosting their own on-premises mitigation systems. This approach necessitates significant investment to effectively combat the constantly evolving attacks, with capacity being finite in the face of escalating attack sizes.

Reviewer: ```suggestion
Internet service providers (ISPs) and telecommunications companies (such as T-Mobile or British Telecom) are vulnerable to network DDoS attacks, which are often focused on the end customer - for example, trying to attack a company or remote worker connected to the internet via a broadband internet service provider. Historically to protect these customers, service providers have relied on hosting their own on-premises mitigation systems. This approach necessitates significant investment to effectively combat the constantly evolving attacks, with capacity being finite in the face of escalating attack sizes.
```
